### ISSUE TITLE
[BUG] Fresh install of 3.2.0-beta leads to 500 after submitting /install

### ISSUE THREAD

**Issue by CantoneseCat** [Leantime/leantime#2572]
**What set up are you using**
* [ ] Cloud Hosted Version
* [x] Self Hosted

**Describe the bug**
When submitting the form on /install on a clean install of 3.2.0-beta, this exception is thrown:
```
PHP Fatal error:  Uncaught TypeError: Cannot assign true to property Leantime\Core\Language::$ini_array of type array in /var/www/virtual/redacted/team/app/Core/Language.php:172"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "Stack trace:"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#0 /var/www/virtual/redacted/team/app/Core/Language.php(136): Leantime\Core\Language->readIni()"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#1 /var/www/virtual/redacted/team/app/Core/Language.php(102): Leantime\Core\Language->setLanguage('en-US')"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#2 [internal function]: Leantime\Core\Language->__construct(Object(Leantime\Core\Environment), Object(Leantime\Core\ApiRequest))"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#3 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(952): ReflectionClass->newInstanceArgs(Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#4 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(293): Illuminate\Container\Container->build('Leantime\\Core\\L...')"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#5 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(908): Illuminate\Container\Container->Illuminate\Container\{closure}(Object(Leantime\Core\Application), Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#6 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(795): Illuminate\Container\Container->build(Object(Closure))"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#7 /var/www/virtual/redacted/team/app/Core/Application.php(471): Illuminate\Container\Container->resolve('Leantime\\Core\\L...', Array, true)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#8 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(731): Leantime\Core\Application->resolve('Leantime\\Core\\L...', Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#9 /var/www/virtual/redacted/team/app/Core/Application.php(456): Illuminate\Container\Container->make('Leantime\\Core\\L...', Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#10 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(1066): Leantime\Core\Application->make('Leantime\\Core\\L...')"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#11 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(982): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#12 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(943): Illuminate\Container\Container->resolveDependencies(Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#13 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(795): Illuminate\Container\Container->build('Leantime\\Core\\T...')"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#14 /var/www/virtual/redacted/team/app/Core/Application.php(471): Illuminate\Container\Container->resolve('Leantime\\Core\\T...', Array, true)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#15 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(731): Leantime\Core\Application->resolve('Leantime\\Core\\T...', Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#16 /var/www/virtual/redacted/team/app/Core/Application.php(456): Illuminate\Container\Container->make('Leantime\\Core\\T...', Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#17 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(1066): Leantime\Core\Application->make('Leantime\\Core\\T...')"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#18 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(982): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#19 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(943): Illuminate\Container\Container->resolveDependencies(Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#20 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(795): Illuminate\Container\Container->build('Leantime\\Core\\T...')"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#21 /var/www/virtual/redacted/team/app/Core/Application.php(471): Illuminate\Container\Container->resolve('Leantime\\Core\\T...', Array, true)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#22 /var/www/virtual/redacted/team/vendor/illuminate/container/Container.php(731): Leantime\Core\Application->resolve('Leantime\\Core\\T...', Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#23 /var/www/virtual/redacted/team/app/Core/Application.php(456): Illuminate\Container\Container->make('Leantime\\Core\\T...', Array)"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#24 /var/www/virtual/redacted/team/app/Core/HttpKernel.php(86): Leantime\Core\Application->make('Leantime\\Core\\T...')"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#25 /var/www/virtual/redacted/team/app/Core/Bootloader.php(161): Leantime\Core\HttpKernel->handle(Object(Leantime\Core\IncomingRequest))"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#26 /var/www/virtual/redacted/team/app/Core/Bootloader.php(139): Leantime\Core\Bootloader->handleRequest()"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#27 /var/www/virtual/redacted/team/public/index.php(18): Leantime\Core\Bootloader->boot()"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "#28 {main}"
[06-Jul-2024 18:36:46] WARNING: [pool www] child 13766 said into stderr: "  thrown in /var/www/virtual/redacted/team/app/Core/Language.php on line 172"

```

**To Reproduce**
1. `cd subdomainDirectory`
2. `wget https://github.com/Leantime/leantime/releases/download/v3.2.0-beta/Leantime-v3.2.0-beta.tar.gz`
3. `tar -xf Leantime-v3.2.0-beta.tar.gz`
4. `mv leantime/{.,}* ./`
5. `mv config/sample.env config/.env`
6. `nano config/.env` Configure database credentials
7. `nano .htaccess` adding `RewriteBase /`
8. `nano public/.htaccess` adding `RewriteBase /public`
9. visit subdomain.domain.tld/install
10. enter email, pw, name, org
11. click `Install` button
12. Receive http error 500

**Expected behavior**
Exception shouldn't be thrown

**Leantime Version**
3.2.0-beta

**Server**
Apache httpd behind nginx on a shared hoster (uberspace.de)

**PHP / MySQL Version**
PHP 8.3.8 (cli) (built: Jun  4 2024 14:53:17) (NTS gcc x86_64)
mysql  Ver 15.1 Distrib 10.6.18-MariaDB, for Linux (x86_64) using readline 5.1

**Additional context**
-

**Comment by VanillaIce24** [comment]
Got the same problem with this and the earlier versions. I have a older 3.1 version that seems to work. On never version I get also these 500 errors.

**Comment by CantoneseCat** [comment]
Seems like this is fixed by #2578 in beta-2

**Comment by marcelfolaron** [comment]
Thanks for confirming!
